corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -21,6 +21,8 @@
     var show_inactive = {{ show_inactive|yesno:"true,false" }};
     var alert_user = hqImport("hqwebapp/js/alert_user").alert_user;
```

Comment:
I think this is only ever used inside `LocationModel` now. This is conceptually an import from another file—you can fix a bunch of those stickler errors that have to do with this (`alert_user`) being a global by moving this line into the appropriate place in the js file.